### ISSUE TITLE
onboard_do_3nic.tpl generates an unusable configuration

### ISSUE THREAD

**Issue by chen23** [f5devcentral/terraform-aws-bigip-module#8]
I've run into two issues using the DO declaration that is generated by onboard_do_3nic.tpl from the 

1. quote on nameserver/dns server is not valid JSON
1. DNS server is not valid for AWS

issue 1: quote on nameserver

error
```
[
  {
    "id": "d920cd82-c730-44f0-9b88-a00ef395c9b3",
    "selfLink": "https://localhost/mgmt/shared/declarative-onboarding/task/d920cd82-c730-44f0-9b88-a00ef395c9b3",
    "code": 400,
    "status": "ERROR",
    "message": "bad declaration",
    "errors": [
      "Unable to parse request body. Should be JSON format."
    ],
```
fix 
```
index 27def98..7e66625 100644
--- a/onboard_do_3nic.tpl
+++ b/onboard_do_3nic.tpl
@@ -12,7 +12,7 @@
     "myDns": {
       "class": "DNS",
       "nameServers": [
-        "${name_servers}"
+        ${name_servers}
       ],
       "search": [
         "f5.com"
@@ -21,7 +21,7 @@
     "myNtp": {
       "class": "NTP",
       "servers": [
-        "${ntp_servers}"
+        ${ntp_servers}
       ],
       "timezone": "UTC"
     },
@@ -70,4 +70,4 @@
       "trafficGroup": "traffic-group-local-only"
     }
   }
-}
\ No newline at end of file
+}
```

issue 2: bad DNS

error
```
  {
    "id": "17c54822-05de-4168-86c7-a7d05cfd0a5a",
    "selfLink": "https://localhost/mgmt/shared/declarative-onboarding/task/17c54822-05de-4168-86c7-a7d05cfd0a5a",
    "code": 424,
    "status": "ERROR",
    "message": "invalid config - rolled back",
    "errors": [
      "tryUntil: max tries reached: Unable to resolve host 2.pool.ntp.org: getaddrinfo ENOTFOUND 2.pool.ntp.org",
      "tryUntil: max tries reached: Unable to resolve host 2.pool.ntp.org: getaddrinfo ENOTFOUND 2.pool.ntp.org"
    ],
```

workaround (remove DNS block)

```
index 27def98..59cae6d 100644
--- a/onboard_do_3nic.tpl
+++ b/onboard_do_3nic.tpl
@@ -9,19 +9,10 @@
       "class": "System",
       "hostname": "${hostname}"
     },
-    "myDns": {
-      "class": "DNS",
-      "nameServers": [
-        "${name_servers}"
-      ],
-      "search": [
-        "f5.com"
-      ]
-    },
     "myNtp": {
       "class": "NTP",
       "servers": [
-        "${ntp_servers}"
+        ${ntp_servers}
       ],
       "timezone": "UTC"
     },
@@ -70,4 +61,4 @@
       "trafficGroup": "traffic-group-local-only"
     }
   }
-}
\ No newline at end of file
+}
```

**Comment by nmenant** [comment]
INFRAANO-121 has been created to update this properly. We will leverage the AWS Internal DNS and NTP 
DNS: 169.254.169.253
NTP: 169.254.169.123

**Comment by papineni87** [comment]
Issue fixed in v0.9.1,try with latest master
